### PR TITLE
Features documentation

### PR DESCRIPTION
**Pull request by agriffard**
List all the features in a new tab of the documentation, organized by group.
Ex: Core, Content, Admin, Design, Extensibility.

### CODE REVIEW

**Comment by agriffard** [comment]
https://docs.orchardcore.net/en/ag-features/docs/features/core/

Each feature should have a small description and a 'See more' line if available somewhere else in the documentation.

We can also add a screenshot if we can find a good image that represents the feature.

Each group should not have too many features (the right TOC should not have to be scrolled to see all the links).
If they are too many, they should be splitted in different groups.

**Comment by Piedone** [comment]
Sorry for taking so much time, I'll review this and https://github.com/OrchardCMS/OrchardCore/pull/7997 soon.

File: src/docs/features/README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Features
```
Comment (Piedone) [inline src/docs/features/README.md:1]: I'm not sure I understand the goal with this new section. Isn't it largely overlapping with the Glossary and Reference sections?

Maybe the Reference section could get a landing page (now it just opens the first page, Content Types) that explains something along the lines of:

```
This is a comprehensive reference for the modules and their features available in Orchard Core. Under each of these pages, you'll find an overview of the given module with documentation on how to use and extend them.
```

But I think providing a categorization of the features based on their areas of function, or goals a user may want to achieve with them is great. So probably also have a similar TOC there as under the "Sections" title here below, and list all related features under the bullet points, linking to their reference pages?

And all the one-sentence summaries under each feature here could be moved to the reference pages, if something like this is missing for the given module.

If you agree, I can create a PR for this.

**Review by Piedone — CHANGES_REQUESTED** [review]

**Comment by agriffard** [comment]
I just want that people get an overview of which features are available.
I am not sure of what is the best way.

The references stick to the structure of the projects and each project/module can have many features so I tried to organize them like the admin sections.

**Comment by agriffard** [comment]
In the end, I want something that is exhaustive (list all current features) and that we keep maintained when we add new features (and may be announce them when planned in milestones).

**Comment by Piedone** [comment]
Yeah, I see that but I don't think it's good to have a third section where people can basically get an overview of what's in the platform (also thinking about these getting out of sync over time).

Right now, this new section is a categorized list of features. But we already have an up to date list of features, so I'd just categorize that (with a TOC that provides a different approach than the structure of the solution folders). And if further documentation is warranted for each module (feature) I'd also continue to add it there.

**Comment by agriffard** [comment]
The features page is not really up to date: https://docs.orchardcore.net/en/dev/docs/reference/modules/Features/
It should be in another page with links to doc page if available.
The version of OC is not really essential.

**Comment by Piedone** [comment]
I don't really see the value of having that page you linked to begin with and it's not really part of OrchardCore.Features either. All in all, I'd still do what I suggested above.

**Comment by agriffard** [comment]
Ok, please do as you suggest @Piedone.

**Pull request closed without merging**